pkg/minishift/provisioner/minishift_provisioner.go

```diff
@@ -88,6 +88,41 @@ func (provisioner *MinishiftProvisioner) Package(name string, action pkgaction.P
 	return errors.New("Custom provisioner does not support package")
 }
 
+func (provisioner *MinishiftProvisioner) remoteHostPrepration() error {
+	packageList := map[string]string{
+		"firewalld": "firewall-cmd",
+		"docker":    "docker",
+		"net-tools": "netstat"}
+
+	// Install required packages if not present.
+	for pkg, cmd := range packageList {
+		if out, err := provisioner.SSHCommand(fmt.Sprintf("which %s || sudo yum install -y %s", cmd, pkg)); err != nil {
```

Comment:
Is this fine for Fedora? as we use the same provisioner for Fedora also, right?

Reply:
`yum` is still a fallback option